modules/docs/sc_param.md

```diff
@@ -36,76 +36,76 @@ The sc_param module provides methods to help you handle parameters for your stre
+| max_buffer_age                          | number | 5                                                                             | if no new event has been stored in the buffer in the past 5 seconds, all stored events are going to be sent even if the max_buffer_size hasn't been reached                                                                                                                                                                                   |                                                                                                                                                                                                                                                                                          |                                                                                                                                                                                                                                                                                                  |
+| max_all_queues_age                      | number | 300                                                                           | if last global flush date was 300 seconds ago, it will force a flush of each queue                                                                                                                                                                                                                                                            |                                                                                                                                                                                                                                                                                          |                                                                                                                                                                                                                                                                                                  |
```

Comment:
```suggestion
| max_all_queues_age                      | number | 300                                                                           | if the last global flush date was 300 seconds ago, it will force a flush of each queue                                                                                                                                                                                                                                                            |                                                                                                                                                                                                                                                                                          |                                                                                                                                                                                                                                                                                                  |
```